src/style.css

```diff
@@ -1 +1,115 @@
-/* add styles here */
+.html {
+  margin: 0;
+  padding: 0;
+}
```

Comment:
It looks like there is a typo in the selector `.html`. It should be `html` to correctly select the root HTML element.